services/jujucharms-com.yaml

```diff
@@ -0,0 +1,51 @@
+---
+
+kind: Service
+apiVersion: v1  # See https://bit.ly/2KdOtrZ
+metadata:
+  name: jujucharms-com
+spec:
+  selector:
+    app: jujucharms.com
+  ports:
+    - name: http
+      protocol: TCP
+      port: 80
+      targetPort: http
+
+---
+
+kind: Deployment
+apiVersion: extensions/v1beta1  # See https://bit.ly/2KdOtrZ
+metadata:
+  name: jujucharms-com
+spec:
+  replicas: 5
+  template:
+    metadata:
+      labels:
+        app: jujucharms.com
+    spec:
+      containers:
+        - name: jujucharms-com
+          image: prod-comms.docker-registry.canonical.com/jujucharms.com:${TAG_TO_DEPLOY}
+          ports:
+            - name: http
+              containerPort: 80
+          env:
```

Comment:
Let's remove this block for now. This site doesn't yet have a sentry key, so this can be added in a follow-up PR sometime.

Reply:
Done